### DISCUSSION TITLE
How often does the integration update?

### DISCUSSION THREAD

**Discussion by Yorkie71** [robbrad/UKBinCollectionData#423] [Q&A] [answered]
Firstly, this is a really useful integration so thanks for the hard work in providing it.

I have a general question about when the integration pulls the data from the council site. Sometimes on the day of collection, the display is still stating "tomorrow". If I reload the integration, then it generally updates so this has prompted my question. Is it also configurable?

TIA.

**Comment by robbrad** [comment, marked as answer]
Not currently configurable - the update is set at every 12 hours to protect council infrastructure.

See: https://github.com/robbrad/UKBinCollectionData/blob/master/custom_components/uk_bin_collection/sensor.py#L127

I think you may have found a bug though - as the display should change but the collection of the data should stay at 12 hours.

Let me take a look at this

@OliverCullimore / @dp247  FYI

**Reply by Yorkie71** [reply]
Thanks for the reply. The 12 hour frequency makes perfect sense and is perfectly adequate. As for the potential bug, do you want me to submit it as an issue?

**Reply by robbrad** [reply]
Yes please 👍

**Reply by Yorkie71** [reply]
[Done](https://github.com/robbrad/UKBinCollectionData/issues/425#issue-1985369208). Thanks for the support Rob. I wasn't able to add the "bug" label for some reason.

**Reply by dp247** [reply]
Gotcha covered :)
